### DISCUSSION TITLE
Small Group Health Plans

### DISCUSSION THREAD

**Discussion by Ginger228** [CMSgov/price-transparency-guide#150] [Q&A] [answered]
Suppose a group health plan doesn’t have an internet site to disclose the three machine-readable files publicly by 1/1/22. Could a third party, clearinghouse, or claims administrator post the files to their website to comply with the rule?

**Comment by harntony** [comment, marked as answer]
Regarding the allowed amount file (out of network) the mandate states that  " Nothing in this section prevents the Allowed Amount File from being hosted on a third-party website or prevents a plan administrator or issuer from contracting with a third party to post the file. However, if a plan or issuer chooses not to also host the file separately on its own website, it must provide a link on its own public website to the location where the file is made publicly available."

**Comment by shaselton-usds** [comment]
@harntony is correct. If you can point people to where the files are being housed/posted, that will work.

**Reply by monty3301** [reply]
Is this also true for the in-network file?  Can employers link to the TPA's site where file is housed?

**Reply by nhartmanNWA** [reply]
If the group health plan does not have it's own dedicated site and the TPA does not want to list all of their clients on their site, would a site that has the group health plan name on the site (thus searchable on the internet) and links to the MRFs suffice?
